client/my-sites/upgrades/checkout/checkout.jsx

```diff
@@ -130,9 +130,13 @@ module.exports = React.createClass( {
 
 	getCheckoutCompleteRedirectPath: function() {
 		var renewalItem;
+
 		if ( cartItems.hasRenewalItem( this.props.cart ) ) {
 			renewalItem = cartItems.getRenewalItems( this.props.cart )[ 0 ];
+
 			return purchasePaths.managePurchaseDestination( renewalItem.extra.purchaseDomain, renewalItem.extra.purchaseId, 'thank-you' );
+		} else if ( cartItems.hasFreeTrial( this.props.cart ) ) {
+			return `/plans/${ this.props.sites.getSelectedSite().slug }/thank-you`;
```

Comment:
I think this will be fine - we don't show checkout during signup - they have to have a site to see checkout.